iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -37,6 +38,12 @@ public class TemperatureControllerSample
         // Dictionary to hold the max temperature since last reboot, for each "Thermostat" component.
         private readonly Dictionary<string, double> _maxTemp = new Dictionary<string, double>();
 
+        // A safe initial value for caching the writable properties version is 1, so the client
+        // will process all previous property change requests and initialize the device application
+        // after which this version will be updated to that, so we have a high water mark of which version number
+        // has been processed.
+        private static long s_localWritableProperties = 1;

```

Comment:
I think it may be worth including the word `Version` in the name. 
something like: `s_localWritablePropertiesVersion`

Reply:
Agree, corrected.